### ISSUE TITLE
Lockscreen and Widget

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#91]
```
Guys, I really want to support this app development and have been looking for a free
alternative to the Logitech official apps for ages, but unless I'm missing something,
Squeezer seems to be missing the two key things that are missing from the official
app: a lockscreen widget and normal screen widget. Implementing these features would
really make Squeezer superior to the official app. Thanks for considering.
```

Original issue reported on code.google.com by `jefftbrown` on 2014-04-08 01:57:56

**Comment by kaaholst** [comment]
```
Hi. 1.3.0 will contain enhanced notification support. On all devices this will allow
you to control playback from the notification. On Lollipop (and, I think, KitKat) devices
you will be able to access these from the lock screen.

This is in beta test now. If you're not in the beta test programme and want to join,
please follow the instructions at https://github.com/nikclayton/android-squeezer/wiki/Beta-testing.
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2015-05-10 19:55:03

**Comment by kaaholst** [comment]
Home screen widgets are available from 2.2.0, so closing this

Original issue reported on github.com/nikclayton/android-squeezer by kaaholst on 2020-09-17T22:05:23Z
